guides/workload-autoscaling/workload-autoscaling/values.yaml

```
@@ -0,0 +1,55 @@
+wva:
+  enabled: true
+
+  image:
+    repository: ghcr.io/llm-d/workload-variant-autoscaler
+    tag: latest
+  imagePullPolicy: Always
+
+  metrics:
+    enabled: true
+    port: 8443
+    secure: true
+  
+  reconcileInterval: 60s
+    
+  prometheus:
+    monitoringNamespace: openshift-user-workload-monitoring # Namespace for Prometheus monitoring for OpenShift 
+    baseURL: "https://thanos-querier.openshift-monitoring.svc.cluster.local:9091"
+    # Development security configuration (relaxed for easier development)
+    tls:
+      insecureSkipVerify: true   # Development: true, Production: false
+      caCertPath: "/etc/ssl/certs/prometheus-ca.crt"
+    # caCert: |  # Uncomment and provide your CA certificate
+    #   -----BEGIN CERTIFICATE-----
+    #   YOUR_CA_CERTIFICATE_HERE
+    #   -----END CERTIFICATE-----
+
+  # Environment variable to enable experimental hybrid-based optimization
+	## When "on", runs both capacity analyzer and model-based optimizer with arbitration
+	## When "model-only" runs model-based optimizer only
+	## When "off" or unset, runs capacity analyzer only (default, reactive mode)
+  experimentalHybridOptimization: false  # Enable experimental hybrid optimization (default: false)
+  scaleToZero: false  # Enable scaling variants to zero replicas (default: false)
+
+llmd:
+  namespace: llm-d-inference-scheduler
+  modelName: ms-inference-scheduling-llm-d-modelservice
+  modelID: "Qwen/Qwen3-0.6B"
+
+va:
+  enabled: true
+  accelerator: H100
+  sloTpot: 10
+  sloTtft: 1000
+
+hpa:
+  enabled: true
+  maxReplicas: 10
+  targetAverageValue: "1"
+
+vllmService:
+  enabled: true
```

This should be disabled - the modelservice chart brings its own decode service + metrics through a podmonitor:

```suggestion
  enabled: false
```